app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from os import path
+import sys
+
+
+def copy_file(command: str) -> str or None:
+    cmd, main_file, new_file = command.split()
+    source = path.join(sys.path[0], main_file)
+    destination = path.join(sys.path[0], new_file)
```

Comment:
there's a function `os.getcwd()` that would turn the current working directory, that, in some cases could be changed prior to the module where your `sys.path[0]` call is located. Also, that function would, arguably, improve your code readability